packages/driver/cypress/integration/issues/7088_spec.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+describe('slatejs', () => {
+  it('test', () => {
+    cy.visit('http://www.slatejs.org/examples/plaintext')
```

Comment:
I know this is in draft, so you may be planning to update, but we don't want any tests that rely on 3rd party hosted domains since our tests would fail if slatejs is ever down.

Reply:
But the bundled file would be huge (react + slate things). Can I use them, then? Until now, there was no case like this. 

Or I might need to create a simplified test file that only check if `beforeinput` event is dispatched and noUpdate works correctly. But it cannot check if it works with slatejs.

Reply:
Yah, we do have some tests testing libraries in general where we just include them in here. But, yah ideally something not so large but testing the behavior. We definitely can't visit a 3rd party url.

Reply:
I compiled and minified [the `plaintext` example.](https://www.slatejs.org/examples/plaintext) The result js file size is 269.9kB. It's a bit smaller than jquery(285.2kB).